src/net.cpp

```diff
@@ -2343,7 +2343,7 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
     threadMessageHandler = std::thread(&TraceThread<std::function<void()> >, "msghand", std::function<void()>(std::bind(&CConnman::ThreadMessageHandler, this)));
 
     // Dump network addresses
-    scheduler.scheduleEvery(std::bind(&CConnman::DumpAddresses, this), DUMP_PEERS_INTERVAL * 1000);
+    scheduler.scheduleEvery([&] { this->DumpAddresses(); }, DUMP_PEERS_INTERVAL);
```

Comment:
```suggestion
    scheduler.scheduleEvery([this] { DumpAddresses(); }, DUMP_PEERS_INTERVAL);
```
Same below.

Reply:
> Same below.

Where exactly?

Reply:
[src/net_processing.cpp:1127](https://github.com/bitcoin/bitcoin/pull/18289/files#diff-eff7adeaec73a769788bb78858815c91R1127)?

Reply:
`consensusParams` needs to be passed in